tests/test_main.py

```diff
@@ -0,0 +1,164 @@
+import os
+import pytest
+import json
+import websocket
+from haas_websocket import main
+from haas_websocket.haas_rest_handler import TokenAuth
+from unittest.mock import MagicMock, patch, call, Mock
+from google.cloud import secretmanager
+
+#--------------------------------------------------------------------------------unit tests for rest sign in module--------------------------------------------------------------------------------
+
+@patch('google.cloud.pubsub_v1.PublisherClient')
+@patch('haas_websocket.haas_rest_handler.TokenAuth')
+@patch.dict(os.environ, {
+    'PROJECT_ID': 'PROJECT_ID',
+    'POINT_TOPIC': 'POINT_TOPIC',
+    'HAAS_WSS_ENDPOINT': 'wss.testwebsocket'
+})
+def test_main_rest_sign_in(pubsub, mTockenAuth):
+    main.TokenAuth = MagicMock(return_value = mTockenAuth)
+    main.restSignIn()
+    mTockenAuth.signIn.assert_called_with()
+
+#--------------------------------------------------------------------------------unit tests for filter module--------------------------------------------------------------------------------
+
+@patch('google.cloud.pubsub_v1.PublisherClient')
+@patch.dict(os.environ, {
+    'PROJECT_ID': 'PROJECT_ID',
+    'POINT_TOPIC': 'POINT_TOPIC'
+})
+def test_main_filter_point(pubsub):
+    message = '{"type":"point"}'
+    main.filterMessage(message,pubsub)
```

Comment:
On these filter tests, let's add a return object and test those values as well so we're validating the return type is as expected. So for this test we could have something like this: 
![image](https://user-images.githubusercontent.com/2932159/157769904-01c4da08-72fb-45ba-bacd-e0923de81940.png)